ezGroceriesShoppingList/src/main/java/com/ezgroceries/shoppinglist/service/CocktailServiceImpl.java

```diff
@@ -0,0 +1,59 @@
+package com.ezgroceries.shoppinglist.service;
+
+import com.ezgroceries.shoppinglist.entity.Cocktail;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class CocktailServiceImpl implements CocktailService {
+
+    @Override
+    public List<Cocktail> getCocktails(String search) {
```

Comment:
Consistency is important, based on the naming convention you have chosen this method should be called findCocktails not getCocktails